hostscripts/soc-ci/soc-ci

```diff
@@ -37,8 +37,8 @@ from six.moves import input
 
 # path to the user configuration file
 USER_CONFIG_PATH = os.path.expanduser('~/.soc-ci.ini')
-# possible mkcX hosts
-CI_WORKERS = list('abcdefghijklmnp')
+# possible mkcX hosts (there is no 'j' host)
+CI_WORKERS = list('abcdefghiklmnp')
```

Comment:
o is missing, though o and p are not used for jenkins atm, just for manual clouds.
and h is not configured for jenkins atm either. Would that matter?